src/mbgl/renderer/render_style.cpp

```diff
@@ -309,15 +309,12 @@ RenderData RenderStyle::getRenderData(MapDebugOptions debugOptions, float angle)
             continue;
         }
 
-        auto& renderTiles = source->getRenderTiles();
         const bool symbolLayer = layer->is<RenderSymbolLayer>();
 
         // Sort symbol tiles in opposite y position, so tiles with overlapping
```

Comment:
We still do this sorting, right, just not here anymore? Can this comment move to where we actually do the sorting?